src/regex_patterns.rs

```diff
@@ -187,4 +189,15 @@ impl crate::parser::Phonetic {
             .collect::<BTreeMap<_, _>>();
         crate::parser::Phonetic { patterns }
     }
+
+    pub fn convert_regex(&self, raw_input: &str) -> String {
+        let input: String = raw_input.to_ascii_lowercase().chars().filter(|&ch| !IGNORE.contains(ch)).collect();
+        let mut output = String::with_capacity(input.len() * 60);
+        output.push('^');
+
+        self.convert_internal(&input, &mut output, "(্[যবম])?(্?)([ঃঁ]?)");
+
+        output.push('$');
```

Comment:
Small styling opinion. I think the empty lines should be before and after push here, not between them. 
```suggestion

        output.push('^');
        self.convert_internal(&input, &mut output, "(্[যবম])?(্?)([ঃঁ]?)");
        output.push('$');

```